apps/wallet/browser/bridge/utils.ts

```diff
@@ -2,7 +2,7 @@ import { logger } from '@walless/core';
 import type { PureMessagePayload } from '@walless/messaging';
 import { createEncryptionKeyVault, createMessenger } from '@walless/messaging';
 import type { SettingDocument } from '@walless/store';
-import { storage } from 'utils/storage';
+import { storage } from 'utils/storage/db';
```

Comment:
This bridge is used for web app communication, can we use a separate implementation to decouple code from another place?